src/main/deltachat.js

```diff
@@ -26,12 +25,16 @@ class DeltaChatController extends events.EventEmitter {
     this._saved = saved
   }
 
+  getPath (addr) {
+    return path.join(this.cwd, Buffer.from(addr).toString('hex'))
+  }
+
   /**
    * Dispatched when logging in from Login
    */
   login (credentials, render, coreStrings) {
     // Creates a separate DB file for each login
-    const cwd = path.join(this.cwd, Buffer.from(credentials.addr).toString('hex'))
+    const cwd = this.getPath(credentials.addr)
```

Comment:
this finally also makes src/main/deltachat.js entirely separate and independent from `electron` which gets us closer to the grand extension idea.. @hpk42